packages/connect-disputable-voting/src/__test__/thegraph/votes.test.ts

```diff
@@ -72,14 +72,14 @@ describe('DisputableVoting votes', () => {
     })
 
     test('allows fetching a single vote', () => {
-      const vote = votes[6]
+      const vote = votes[2]
```

Comment:
Should we do [something like this](https://github.com/aragon/connect/commit/fd67970e75f462cde9858e10a5ba719f4f6fe065) to prevent this to break when votes get added?

Reply:
Interesting, note that in this PR introduced a sorting instruction for the query, then votes should be ordered, but I see your point definitely